lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -513,5 +513,221 @@ module Resources
+            'performer': [
+              {
+                'function': {
+                  'coding': [
+                    {
+                      'system': 'http://terminology.hl7.org/CodeSystem/v2-0912',
+                      'code': 'OP',
+                      'display': 'Ordering Provider'
+                    }
+                  ],
+                  'text': 'Ordering Provider'
+                },
+                'actor': {
+                  'reference': 'Practitioner/190182805'
+                }
+              }
+            ],
+            'reason': [
+              {
+                'extension': [
+                  {
+                    'valueUnsignedInt': 1,
+                    'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/priority'
+                  }
+                ],
+                'coding': [
+                  {
+                    'extension': [
+                      {
+                        'valueCoding': {
+                          'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                          'code': '3693'
+                        },
+                        'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule'
+                      }
+                    ],
+                    'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD',
+                    'code': 'Y22'
+                  }
+                ]
+              }
+            ],
+            'extension': [
+              {
+                'extension': [
+                  {
+                    'valueCodeableConcept': {
+                      'coding': [
+                        {
+                          'extension': [
+                            {
+                              'valueCoding': {
+                                'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                                'code': '3692'
+                              },
+                              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule'
+                            }
+                          ],
+                          'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-MODIFIER',
+                          'code': '26'
+                        }
+                      ]
+                    },
+                    'url': 'code'
+                  },
+                  {
+                    'valueUnsignedInt': 1,
+                    'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/priority'
+                  }
+                ],
+                'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/modifier-code'
+              },
+              {
+                'valueReference': {
+                  'reference': 'Location/683731',
+                  'display': '1 West SA2'
```

Comment:
do we really support taking in a display here?

Reply:
@alexbeckner1 In modify also display is present https://github.com/cerner/fhir.cerner.com/pull/780/files#diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR608-R612 so do we need to remove from the create.